README.md

```diff
@@ -31,6 +31,14 @@ To start the terminal cast application, run:
 ```
 python3 -m cast
 ```
+#### Docker
+Build image
+
+`docker build -t cast .`
```

Comment:
Can you please use ``` instead of just  ` ?
So, we'll have 
```
this
```
instead of 
`this`

Reply:
yep, no problem! Sorry about not seeing that convention in the first place.